### PR TITLE
DOR-2479: Make it work outside GOPATH with custom package path

### PR DESCRIPTION
**Pull request by fraenky8**
This way we can handle non Go-conform project paths like
`User/name/Go/Dorameon` instead of
`User/name/Go/github.com/zalora/doraemon`,

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 language: go
 
 go:
+  - 1.15
```
Comment (zhangilishbay) [inline .travis.yml:4]: What is this file responsible for? 🤔

Comment (fraenky8) [reply .travis.yml:4]: It's the config file for travis CI (see the checks for this PR, they passed).

**Comment by fraenky8** [comment]
Closing this PR as we do not want to customize/touch this tool any further. We will stick to convention over configuration here. This said when cloning Doraemon we need to have it in the directory structure of `github.com/zalora/doraemon`.

**Comment by fraenky8** [comment]
Reopening this PR as we encounter multiple cases for usage of Doraemon outside GOPATH. (Eg using the ~/doraemon created by shop-docker).

**Comment by zhangilishbay** [comment]
> Reopening this PR as we encounter multiple cases for usage of Doraemon outside GOPATH. (Eg using the ~/doraemon created by shop-docker).

Can we also add this comment to the ticket to make it super transparent? 👍 🙇

**Review by zhangilishbay — APPROVED** [review]
LGTM 👍

**Pull request merged**